agent/tool-scripts/datalog/disk-datalog

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+interval=$1
+file=$2
+
+while true; do
+	# we display the output in kilobytes.
+	/usr/bin/df --output=source,target,fstype,itotal,iused,iavail,size,used,avail,file | pbench-log-timestamp | awk -F ' ' '{gsub(/\:/,"",$1); print}' >> $file
```

Comment:
It would have been nice to drop the header lines here.  The regex in post-process is fragile, as it appears to skip the header line because the `target` column header uses `Mounted on`.